pkg/platform/packet/packet.go

```diff
@@ -190,6 +194,11 @@ func createTerraformConfigFile(cfg *config, terraformPath string) error {
 		return errors.Wrapf(err, "failed to marshal management CIDRs")
 	}
 
+	controllerCLCSnippetsBytes, err := json.Marshal(cfg.ControllerCLCSnippets)
```

Comment:
Idem comment on worker pools, I'd keep them as a slice and render them in the template properly and, IIUC, that should work here too and simpler. Similar to what is done [here](https://github.com/kinvolk/lokomotive/blob/2f92fbe8789dcb49d707a233fe703f017be7721d/pkg/platform/packet/template.go#L79-L85)

Reply:
thanks for the tip. removed the unnecessary logic.